crates/sui-bridge/src/sui_client.rs

```diff
@@ -464,6 +504,60 @@ impl SuiClientInner for SuiSdkClient {
         return Ok(status);
     }
 
+    async fn get_token_transfer_action_onchain_signatures(
+        &self,
+        bridge_object_arg: ObjectArg,
+        source_chain_id: u8,
+        seq_number: u64,
+    ) -> Result<Option<Vec<Vec<u8>>>, BridgeError> {
+        let pt = ProgrammableTransaction {
+            inputs: vec![
+                CallArg::Object(bridge_object_arg),
+                CallArg::Pure(bcs::to_bytes(&source_chain_id).unwrap()),
+                CallArg::Pure(bcs::to_bytes(&seq_number).unwrap()),
+            ],
+            commands: vec![Command::MoveCall(Box::new(ProgrammableMoveCall {
+                package: BRIDGE_PACKAGE_ID,
+                module: Identifier::new("bridge").unwrap(),
+                function: Identifier::new("get_token_transfer_action_signatures").unwrap(),
+                type_arguments: vec![],
+                arguments: vec![Argument::Input(0), Argument::Input(1), Argument::Input(2)],
+            }))],
+        };
+        let kind = TransactionKind::programmable(pt.clone());
+        let resp = self
+            .read_api()
+            .dev_inspect_transaction_block(SuiAddress::ZERO, kind, None, None, None)
```

Comment:
is this code (the bridge client) talking to a fullnode or a validator?
and if validators, do they have devinspect?